jenkins/promotion/release-promotion.jenkinsfile

```diff
@@ -377,46 +428,92 @@ pipeline {
                 echo 'Promotion successful for OpenSearch Linux rpm x64!'
             }
         }
-        stage('OpenSearch Dashboards Linux tar arm64') {
-            agent {
-                docker {
-                    label AGENT_LINUX_X64
-                    image 'docker/library/alpine:3'
-                    registryUrl 'https://public.ecr.aws/'
-                    alwaysPull true
+        stage('Invalidate cloudfront and promote docker'){
+            parallel {
+                stage('Invalidate Cloudfront') {
+                agent {
+                    docker {
+                        label AGENT_LINUX_X64
+                        image 'docker/library/alpine:3'
+                        registryUrl 'https://public.ecr.aws/'
+                        alwaysPull true
+                    }
+                }
+                steps {
+                    script {
+                        echo 'Invalidating cloudfront'
+                        def version=params.RELEASE_VERSION.find(/\d/)
```

Comment:
Can you change this to name `major_version`, or it will be pretty confusing.

Reply:
Done!